pkg/minikube/problem/err_map.go

```diff
@@ -0,0 +1,102 @@
+package problem
+
+import "regexp"
+
+// r is a shortcut around regexp.MustCompile
+func r(s string) *regexp.Regexp {
```

Comment:
maybe call this "re()", or perhaps that is just me mixing it with `r''` raw strings (from [python](https://docs.python.org/2/library/re.html)...)